website/docs/patterns/disabled-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,74 @@
+
+In scenarios where a user lands on a page they cannot access, provide a reason why with guidance so that users understand what steps they can take to rectify the error. So that users don't feel stuck, provide them a call to action to facilitate a path forward.
+
+![Showing a 403 error page with information about what is going on, with a way to go back and a way to log in with a different token](/assets/patterns/disabled-patterns/permissions-example-3.png)
+
+### Conditional availability
+
+When a feature or function is conditionally available depending on other steps that must be taken, provide context in place of that action so that users are able to take those steps to enable this feature.
+
+!!! Dont
+
+In this example, the button is disabled and no context is provided, leaving the user confused and unaware of the steps required to enable the feature.
+
+![Showing an interface with a disabled button that doesn't describe why that button is disabled. This is not a recommended experience.](/assets/patterns/disabled-patterns/conditional-availability-example-disabled.png)
+
+Disabled elements lack interactive states, preventing a user from hovering, focusing, or clicking them. For users with assistive technology (e.g., a screen reader), disabled elements are almost entirely ignored, including the text within.
+
+!!!
+
+
+!!! Do
+
+Instead, we recommended replacing the disabled element with contextual guidance to help users to understand why this feature isn't available, and providing them with clear steps to enable it.
+
+![Showing an interface with a compact alert explaining why the expected behavior isn't there with guidance.](/assets/patterns/disabled-patterns/conditional-availability-example-explanation.png)
+
+If possible, provide CTAs directly to the place where users can take immediate action.
+
+!!!
+
+
+### Upgrading
+
+When premium features are available and a user has the necessary permissions, highlighting a path to upgrade provides a graceful marketing opportunity.
+
+An example of this is Figma's "create branch" function that is only available at the enterprise tier. Clicking "create a branch" in Figma opens a marketing modal that entices users to upgrade to gain access to this feature.
+
+![Screen shot of figma's modal showing information about how to upgrade to enterprise to gain access to creating branches.](/assets/patterns/disabled-patterns/upgrade-modal-example.png)
+
+
+### Service outage
+
+In instances where outages may occur, provide a clear message to users on what they can do and add a link to [our status page](https://status.hashicorp.com/).
+
+An example of this is Instagram's login page. When a user has no connection to their servers, a message will appear on submit.
+
+![Displaying Instagram's login page with an error below notifying the user they have no connection to instagram.](/assets/patterns/disabled-patterns/service-outage-example.png)
```

Comment:
Not trying to be difficult here, but I don't recall us using external examples in our documentation (either good or bad); I would be skeptical about introducing such references, especially as we don't know much about their origins and reasoning.

Reply:
This is a good point, @alex-ju! We've discussed this in Design Review, and @andgen404 is going to try to find some relevant internal examples to use instead.

Reply:
Very good point, I'll swap this out and use a more HashiCorp example :)

Reply:
Replaced with our 500 error example. Resolving.